HymnsTests/Display/DisplayHymnBottomBarViewModelSpec.swift

```diff
@@ -175,10 +172,10 @@ class DisplayHymnBottomBarViewModelSpec: QuickSpec {
                     expect(target.buttons[0]).to(equal(.share("Drink! a river pure and clear that's flowing from the throne;\nEat! the tree of life with fruits abundant, richly grown\n\nDo come, oh, do come,\nSays Spirit and the Bride:\n\n")))
                     expect(target.buttons[1]).to(equal(.fontSize))
                     expect(target.buttons[2]).to(equal(.tags))
-                    expect(target.buttons[3]).to(equal(.songInfo(SongInfoDialogViewModel(hymnToDisplay: classic1151))))
-                    expect(target.buttons[4]).to(equal(.soundCloud(URL(string: "https://soundcloud.com/search?q=title")!)))
-                    expect(target.buttons[5]).to(equal(.youTube(URL(string: "https://www.youtube.com/results?search_query=title")!)))
+                    expect(target.buttons[3]).to(equal(.soundCloud(URL(string: "https://soundcloud.com/search?q=title")!)))
+                    expect(target.buttons[4]).to(equal(.youTube(URL(string: "https://www.youtube.com/results?search_query=title")!)))
                     expect(target.overflowButtons).to(beNil())
+                    expect(target.buttons[5]).to(equal(.songInfo(SongInfoDialogViewModel(hymnToDisplay: classic1151))))
```

Comment:
nit: move this 1 line up so it's group withed the other calls

Reply:
resolved